projects/novo-elements/src/utils/Helpers.ts

```diff
@@ -127,7 +127,7 @@ export class Helpers {
     return obj instanceof Date;
   }
 
-  static convertToArray(obj: undefined | Object | Object[]): any[] {
+  static convertToArray(obj: any): any[] {
```

Comment:
can this be any more type specific?

Reply:
`unknown` might be appropriate here. We don't really use it but after @adrianossi brought it up in UI Community for our convo on when to break typing, I started looking into it, and it's appropriate in a lot of the places where we use `any` to mean "really no idea what this type is and no guarantees can be made about it at runtime". I don't have an informed opinion about whether this is one of those instances, but I thought I'd mention it.

Reply:
I could type it fairly specifically. `static convertToArray(obj:  undefined | IDataTableFilter | IDataTableFilter[] ): [] | IDataTableFilter[] {`

At the moment only my filter functions are using this.

Reply:
Using the previously mentioned types would involve importing them in to the Helpers file, which I don't think I want to do. This file should probably remain as simple as possible. I opted to use the unknown type as the input type.